templates/layout/region--header.html.twig

```diff
@@ -0,0 +1,11 @@
+{% embed "@organisms/site-header/site-header.twig" with {
+  site_name: site_name,
+  site_header__border_thickness: '8',
+} %}
+  {% block site_header__utility_nav %}
+    {{ elements.user_account_menu }}
+  {% endblock %}
+  {% block site_header__primary_nav %}
+    {{ elements.main_navigation }}
+  {% endblock %}
+{% endembed %}
```

Comment:
Do we want to add everything else in the header region? Or do we want to manually add each block to this region? When working together today we saw that alerts added to this region did not appear. I'm not against calling them out by name as site owners will not be able to add things in the block placement interface.

Reply:
As we talked about, we'll leave this as is for now. There's an issue for Drupal 10 with a patch that can work if we decide we need it sooner: https://www.drupal.org/project/drupal/issues/3178202.